src/Language/en/ShieldOAuthLang.php

```diff
@@ -18,6 +18,7 @@
     'Callback' => [
         'oauth_class_not_set' => 'An error occurred, it seems that the OAuth Class is not set.',
         'anti_forgery'        => 'Your request has been detected as fake. we are sorry!',
+        'account_not_found'   => 'Your account not found. Please register first!',
```

Comment:
Please add this key for other languages as well.

Reply:
sorry, I don't understand spelling and grammar well enough for other countries, if you wish I will use the help of Google Translate to do it?

Reply:
```php
'account_not_found'   => '(To be translated) Your account not found. Please register first!',
```
The above is suitable for other languages.

Reply:
OK, I'll update for other languages